addons/web/static/src/js/widgets/colorpicker.js

```diff
@@ -140,11 +165,11 @@ var Colorpicker = Dialog.extend({
      * @param {integer} g
      * @param {integer} b
      */
-    _updateRgb: function (r, g, b) {
+    _updateRgba: function (r, g, b, o) {
         var hex = Colorpicker.prototype.convertRgbToHex(r, g, b);
         if (hex) {
             _.extend(this.colorComponents,
-                {red: r, green: g, blue: b},
+                {red: r, green: g, blue: b, opacity: o},
```

Comment:
opacity is a color component (like you did at all other places), so it should not be in the red green blue object